tools/benchmarking/benchmark.py

```diff
@@ -218,6 +256,8 @@ def sweep(run_config: Union[DictConfig, ListConfig], device: int = 0, seed: int
 
     # Run benchmarking for current config
     model_metrics = get_single_model_metrics(model_config=model_config, openvino_metrics=convert_openvino)
+    print(model_config.model.name, model_config.dataset.category)
```

Comment:
What if we want to run `benchmark.py` for custom datasets that don't have `category` field?